log-replication/src/main/java/org/corfudb/logreplication/infrastructure/GRPCLogReplicationServerHandler.java

```diff
@@ -0,0 +1,136 @@
+package org.corfudb.logreplication.infrastructure;
+
+import io.grpc.stub.StreamObserver;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.infrastructure.CustomServerRouter;
+import org.corfudb.runtime.Messages;
+import org.corfudb.runtime.Messages.CorfuMessage;
+import org.corfudb.runtime.LogReplicationChannelGrpc;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * GRPC Log Replication Service Stub Implementation.
+ *
+ * Note: GRPC is used to test the plugin-based transport architecture for log replication.
+ *
+ * @author annym 05/15/20
+ */
+@Slf4j
+public class GRPCLogReplicationServerHandler extends LogReplicationChannelGrpc.LogReplicationChannelImplBase {
```

Comment:
It is very easy to confuse this with the handlers in the CustomServerRouter.  Can we add comments to explain the role of this handler?  Or rename it accordingly?